src/main/java/com/google/graphgeckos/dashboard/api/components/BuildInfo.java

```diff
@@ -0,0 +1,69 @@
+package com.google.graphgeckos.dashboard.api.components;
```

Comment:
Please change this and all sub-classes to com.google.graphgeckos.dashboard.storage, since it is conceptually a part of the storage component.

Reply:
Controllers and shared daos are not the parts of the storage. I would suggest to add new package with shared daos

Reply:
The POJOs are part of the storage component, since it makes _no_ sense for them existing without the storage itself.